pkg/scheduler/framework/plugins/queuesort/priority_sort.go

```diff
@@ -42,7 +42,8 @@ func (pl *PrioritySort) Name() string {
 func (pl *PrioritySort) Less(pInfo1, pInfo2 *framework.QueuedPodInfo) bool {
 	p1 := corev1helpers.PodPriority(pInfo1.Pod)
 	p2 := corev1helpers.PodPriority(pInfo2.Pod)
-	return (p1 > p2) || (p1 == p2 && pInfo1.Timestamp.Before(pInfo2.Timestamp))
+	return (p1 > p2) || (p1 == p2 &&
```

Comment:
It's not a good idea to change the structure of Timestamp. Per https://github.com/kubernetes/kubernetes/issues/117979#issuecomment-1548689549, I'd like to only refresh InitialAttemptTimestamp (maybe also Timestamp) on the first time a pod is queued into activeQ.

Reply:
Reverted and refreshed the `InitialAttemptTimestamp`.